Bump xgb version

interesaaat opened the pull request:
We were forcing ourselves to `xgb==0.90`. This PR enable later xgb versions.

interesaaat committed:
bump xgb version

interesaaat committed:
fix to check in _utils

ksaur commented:
is `base_prediction` the only change we need? I thought there was some problem with a type change with `alpha`, but maybe that is for operators we haven't ported in yet?

interesaaat commented:
> is `base_prediction` the only change we need? I thought there was some problem with a type change with `alpha`, but maybe that is for operators we haven't ported in yet?

Yes that's the only change. We renamed `alpha` into `base_prediction` when we refactored the trees (I think the term is more generic, it was not clear what "alpha" means).
